app/jobs/dashboard_stats_job.rb

```diff
@@ -4,10 +4,10 @@ class DashboardStatsJob < ApplicationJob
 
   sidekiq_options unique: :until_executed
 
-  def perform
+  def perform(location_id)
     dashboard_stats = ImmunizationCacheDatum.where(:name => "dashboard_stats").pick(:value)
```

Comment:
How are we getting dashboard stats for the specific site to broadcast on the specific site channel?

Reply:
You should read on the followup logic